FunctionalTableData/Separator.swift

```diff
@@ -21,11 +21,12 @@ public class Separator: UIView {
 	
 	/// The style for table cells used as separators.
 	///
-	/// The options are `full`, `inset`, and `moreInset`
-	public enum Style {
+	/// The options are `full`, `inset`, `moreInset`, `custom`
+	public enum Style: Equatable {
 		case full
 		case inset
 		case moreInset
+		case custom(leftInset: CGFloat, rightInset: CGFloat, layoutMarginsRelative: Bool)
 		
 		public var insetDistance: CGFloat {
```

Comment:
I think we could get rid of this public var instead. It's only used in one spot and I was a bit confused about the .custom being 0 when .custom actually has values associated with it

Reply:
Removed.